specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -6995,6 +7046,153 @@
       },
       "description": "The List Extension operation response"
     },
+    "VirtualMachineSoftwarePatchProperties": {
+      "properties": {
+        "patchId": {
+          "type": "string",
+          "description": "A unique identifier for the patch."
+        },
+        "name": {
+          "type": "string",
+          "description": "The friendly name of the patch."
+        },
+        "version": {
+          "type": "string",
+          "description": "The version number of the patch. This property applies only to Linux patches."
+        },
+        "kbid": {
+          "type": "string",
+          "description": "The KBID of the patch. Only applies to Windows patches."
+        },
+        "classifications": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The classification(s) of the patch as provided by the patch publisher."
+        },
+        "rebootBehavior": {
+          "type": "string",
+          "description": "Describes the reboot requirements of the patch.",
+          "enum": [
+            "NeverReboots",
+            "AlwaysRequiresReboot",
+            "CanRequestReboot"
+          ],
+          "x-ms-enum": {
+            "name": "SoftwareUpdateRebootBehavior",
+            "modelAsString": true
```

Comment:
Will more enum values be added in the future? If not, set modelAsString to false.

Reply:
Possibly. I don't know for certain, but is seems likely that we may want to add new values in the future.